backend/src/main/resources/com/bakdata/conquery/models/events/generation/BucketTemplate.ftl

```diff
@@ -187,27 +187,6 @@ public class Bucket_${suffix} extends Bucket {
 		</#list>
 	} 
 
-	@Override
-	public boolean eventIsContainedIn(int event, Column column, CDateSet dateRanges) {
-		if(!this.has(event, column)) {
-			return false;
-		}
-		switch(column.getPosition()) {
-		<#list imp.columns as col>
-            <#if col.type.lines != col.type.nullLines>
-                <#if col.type.typeId == "DATE">
-                    case ${col.position}:
-                    return dateRanges.contains(<@f.getMajor col/>(event));
-                <#elseif col.type.typeId == "DATE_RANGE">
-                    case ${col.position}:
-                    return dateRanges.intersects(<@f.getMajor col/>(event));
-                </#if>
-            </#if>
-		</#list>
-		default:
-		throw new IllegalArgumentException("Column "+column+" is not a date type");
-		}
-	}
```

Comment:
wurde die nie aufgerufen?

Reply:
doch aber es macht wenig sinn mMn die hier extra zu kompilieren, habe sie zusammen gefasst in Bucket.